src/app/main.ts

```diff
@@ -33,6 +33,10 @@
     jQuery(document).on('renuo-cms-reload', function () {
       controller.init();
     });
+
+    jQuery(document).on('renuo-cms-reload', function (event:Event, htmlContentBlocks:HTMLElement[]) {
```

Comment:
- i would rather prefer to only (re)load a single content block ("Reload single content via trigger")
- as a developer, i would like to call it via a string instead of an html element, where the string is the content key